editor.planx.uk/src/ui/editor/RichTextInput.tsx

```diff
@@ -497,7 +497,7 @@ const RichTextInput: FC<Props> = (props) => {
           tippyOptions={{
             duration: 100,
             // Hack to "stop" transition of BubbleMenu
-            moveTransition: "transform 600s"
+            moveTransition: "transform 600s",
```

Comment:
My husky / pre-commit hook must not be picking these up properly sorry - will take a look!

Reply:
No worries - mine is out of whack sometimes too!